### PR TITLE
Fix invoices not showing on user side

### PR DESCRIPTION
**Pull request by hedudelgado**

### CODE REVIEW

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/organizations_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module MnoEnterprise::Concerns::Controllers::Jpi::V1::OrganizationsController
   extend ActiveSupport::Concern
-
+
```
Comment (alexnoox) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/organizations_controller.rb]: Empty spaces

**Comment by alexnoox** [comment]
@hedudelgado Could you add a spec with an invoice, thanks.

File: api/spec/controllers/mno_enterprise/jpi/v1/organizations_controller_spec.rb [hunk 2]
```diff
@@ -20,7 +20,7 @@ module MnoEnterprise
 
     # Stub organization + associations
     let(:metadata) { {} }
-    let!(:organization) { build(:organization, metadata: metadata, orga_invites: [], users: [], orga_relations: [], credit_card: credit_card) }
+    let!(:organization) { build(:organization, metadata: metadata, orga_invites: [], users: [], orga_relations: [], credit_card: credit_card, invoices: []) }
```
Comment (BrunoChauvet) [inline api/spec/controllers/mno_enterprise/jpi/v1/organizations_controller_spec.rb]: @hedudelgado can you add a test scenario with rendering invoices so it exercises the view rendering?

**Comment by hedudelgado** [comment]
https://github.com/maestrano/mno-enterprise-angular/pull/331

File: api/spec/controllers/mno_enterprise/jpi/v1/organizations_controller_spec.rb (hunk 2 above)
Comment (hedudelgado) [reply api/spec/controllers/mno_enterprise/jpi/v1/organizations_controller_spec.rb]: Hi @BrunoChauvet,
Done!

**Pull request merged**